mjcf_to_sdformat/mjcf_to_sdformat/converters/world.py

```diff
@@ -16,14 +16,16 @@
 
 from mjcf_to_sdformat.converters.light import mjcf_light_to_sdf
 from mjcf_to_sdformat.converters.link import mjcf_body_to_sdf
+from mjcf_to_sdformat.converters.sensor import mjcf_camera_sensor_to_sdf
 
 import sdformat_mjcf_utils.sdf_utils as su
 from sdformat_mjcf_utils.defaults import MjcfModifiers
 
 import sdformat as sdf
 
 
-def mjcf_worldbody_to_sdf(mjcf_root, physics, world):
+def mjcf_worldbody_to_sdf(mjcf_root, physics, world,
+                          export_world_plugins=True):
```

Comment:
I think the default should be False. Or at least match the command line 
default.

Reply:
https://github.com/gazebosim/gz-mujoco/pull/49/commits/99183420d1e0a8fcfd61f02d6cfe1e7427de9790